.eslintrc.json

```diff
@@ -2,7 +2,9 @@
   "root": true,
   "env": {
     "es2021": true,
-    "node": true
+    "node": true,
+    "mocha": true,
+    "jest": true
```

Comment:
you can have both of these???

Reply:
From the [docs](https://eslint.org/docs/user-guide/configuring/language-options#specifying-environments)

> These environments are not mutually exclusive, so you can define more than one at a time.